public/stylesheets/site/2.0/08-actions.css

```diff
@@ -87,7 +87,7 @@ ol.pagination, div.pagination, ol.year {
   color: #111;
 }
 
-.actions a:hover, .actions input:hover, .actions a:focus, .actions input:focus, label.action:hover, .action:hover, .action:focus {
+.actions a:hover, .actions button:hover, .actions input:hover, .actions a:focus, .actions input:focus, label.action:hover, .action:hover, .action:focus {
```

Comment:
Things using `button_to` were previously `input type="submit"` but now are buttons, so this is needed for consistent styling (note: the cursor doesn't change to a pointer on hover, but that's consistent with the original behaviour)

Reply:
To ensure consistency across buttons, inputs, and links, I think we should also have
- `.actions button` before `.actions input` on line 23
- `.actions button:focus` before `.actions input:focus` here on line 90
- `.actions li button` before `.actions li input` on line 114

We'll also need some skin changes. It's probably easier to show you commits than try to write this out 😆
* https://github.com/otwcode/otwarchive/commit/86db6395ee566e31d189a6aa4b7c268b6106b445
* https://github.com/otwcode/otwarchive/commit/02f79c52fdbeac41b1015bdcbb44ca292fb2106e has some bonus focus fixes for Snow Blue because they were annoying me, but I can do it as a separate PR

Reply:
Made [AO3-6767](https://otwarchive.atlassian.net/browse/AO3-6767) so I don't forget the focus fixes.